uint16 position
float32 velocity_feedforward
---
bool success